models/staging/edfi_3/stage/stg_ef3__stu_ed_org__cohort_years.sql

```diff
@@ -0,0 +1,19 @@
+with stage_stu_ed_org as (
+    select * from {{ ref('stg_ef3__student_education_organization_associations') }}
+),
+flattened as (
+    select 
+        tenant_code,
+        api_year,
+        k_student,
+        k_student_xyear,
+        ed_org_id,
+        k_lea,
+        k_school,
+        {{ extract_descriptor('value:cohortYearTypeDescriptor::string') }} as cohort_year,
+        value:schoolYearTypeReference:schoolYear::string as school_year,
+        {{ extract_descriptor('value:termDescriptor::string') }} as term
```

Comment:
can you name this `academic_term` to match other places? https://github.com/search?q=repo%3Aedanalytics%2Fedu_edfi_source%20termdescriptor&type=code